.github/workflows/examples-bl602.yaml

```diff
@@ -55,8 +55,7 @@ jobs:
       - name: Build example BL602 Lighting App
         timeout-minutes: 30
         run: |
-          ./scripts/run_in_build_env.sh \
-            "./scripts/build/build_examples.py --target bl602-light build"
+          scripts/examples/gn_bl602_example.sh lighting-app ./out/bl602-light
```

Comment:
please keep build_examples.py usable.
Having a separate shell script for every platform makes things easy to diverge and is less consistent.

Reply:
Rolled back this change.